x/hub-genesis/keeper/params.go

```diff
@@ -15,19 +15,3 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
```

Comment:
again, Params is now empty, can we not completely get rid of it for this module? Will a migration be needed?

Reply:
Yes you're right it should be removed
but is it ok if I do it in a followup PR? This PR is currently all tested etc
https://github.com/dymensionxyz/dymension-rdk/issues/452
I have made an issue for myself
Thanks